onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py

```diff
@@ -1073,10 +1334,35 @@ def parse_args():
         nargs="+",
         required=False,
         help="Key-value pairs in op_type:axis_to_quantize separated by space."
-        "Specify the axis to quantize for an op. Default {MatMul:0, Gather:1}"
-        "Example: --quant_axes MatMul:0 Gather:1",
+             "Specify the axis to quantize for an op. Default {MatMul:0, Gather:1}"
+             "Example: --quant_axes MatMul:0 Gather:1",
+    )
+    # Additional arguments specific to nvidia_awq
+    parser.add_argument(
+        "--calib_dataset_name",
+        type=str,
+        default="cnn",
+        help="Name of the calibration dataset for nvidia_awq.",
+    )
+    parser.add_argument(
+        "--tokenizer_dir",
+        type=str,
+        required=False,
+        help="Path of the tokenizer dir.",
+    )
+    parser.add_argument(
+        "--calibration_method",
+        type=str,
+        required=False,
+        choices=["awq", "awq_clip"],
```

Comment:
nit: how about the following:
```
default="awq_lite",
choices=["awq_lite", "awq_clip"],
```
Then, there is no need to add extra code to post-process the value later.

Reply:
intentionally kept is like this